include/event_dbus_monitor.hpp

```diff
@@ -136,8 +137,7 @@ inline void hostStatePropertyChange(sdbusplus::message::message& msg)
         if (*type == "xyz.openbmc_project.State.Host.HostState.Off")
         {
             // Reset system dump handler to handle power off scenarios
-            // TODO: Uncomment this resetHandlers after dump offload support
-            // crow::obmc_dump::resetHandlers();
+            crow::obmc_dump::resetHandlers();
```

Comment:
No concern of this causing other problems?

Reply:
there is no problems, it just cleans up sockets and connection